plugins/bazel/README.md

```diff
@@ -0,0 +1,24 @@
+# Bazel Plugin
+
+
+
+## Installation
+
+This plugin is not included with the `auto` CLI installed via NPM. To install:
```

Comment:
It might be worth bundling this into the binary version so that projects that don't have any NPM deps can still consume it easily.

Reply:
Gotcha! I think that was included/set by default when the plugin was scaffolded. I will look into changing that.